### ISSUE TITLE
Strange behavior when adding photos to the list

### ISSUE THREAD

**Issue by nunocgtt** [rvera/image-picker#52]
Hello,

For starters, great plugin.
I have a situation where I'm using bootstrap and your plugin to show a bunch of photos. I have the ability to upload a new photo to that album that is refreshed via ajax.

The problem I have is intermittent (doesn't always happen), but sometimes the pictures are misplaced in the grid (i have a 4xN grid of photos) meaning sometimes a new row will add the first picture in slot no. 2.

![problem](https://cloud.githubusercontent.com/assets/10992211/6186947/6c2bc1a8-b378-11e4-91d9-d99ab198e7c9.png)

As you can see in the image, the second row starts off by 1 picture. Row 3 is ok as well.
I've checked my css classes and nothing unusual is there. The list is <li> for last picture of row 1 and <li> for first picture in row 2 meaning no html no gaps and no padding occur between them.

Thank you for your time.

**Comment by nunocgtt** [comment]
... hello?

**Comment by Mahmoudz** [comment]
I as well have some troubles with this library, it does strange behavior when dynamically adding photos to it. Not only on the CSS side but as well it affects the end results with duplication..
Seems the contributors are not maintaining it anymore!

**Comment by rvera** [comment]
Sorry for the late reply. It was not in the original scope to handle dynamic photos, the current solution for this is to add the new options to the `select` object and call resync on the picker.

It would be great if you could provide a jsfiddle of any issues you are seeing. Thanks.

**Comment by nunocgtt** [comment]
The resync was already being done. 
However the pictures intermitently appear misaligned to the right. Seems a js issue as the html is good clean and consistent.

**Comment by jordan314** [comment]
I got this working, by calling imagepicker() again once I've appended items, and also waiting for imagesLoaded (using desandro's imagesLoaded plugin) and then scrolling to the bottom of the div when they've loaded.
```
$('#profilephotopicker').imagesLoaded(function(){
    $('#profilephotos').scrollTop($('#profilephotos')[0].scrollHeight);
});
```

**Comment by rvera** [comment]
From your original pic, that looks a browser css issue where floating items are not properly placed. 

Will close this for now but would love to hear how you solved this and if you have any generic solutions.
